File: api/src/main/java/com/aiko/api/model/Stop.java
package com.aiko.api.model;

import java.util.List;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.ManyToMany;

import com.googlecode.jmapper.annotations.JMap;

import lombok.Data;

@Data
@Entity
public class Stop {
  
  @Id
  @GeneratedValue(strategy=GenerationType.IDENTITY)
  private long id;

  @JMap
  private String name;

  @JMap
  private Double latitude;
  @JMap
  private Double longitude;

  @ManyToMany(mappedBy = "stops")
  private List<Line> lines;
}

File: TestBackEnd.sln/src/TestBackEnd.Domain/Entities/Stop.cs
using System.Collections.ObjectModel;

namespace TestBackEnd.Domain.Entities
{
    public class Stop : Base
    { 
        public string Name { get; private set; }
        
        public double Latitude { get; private set; }

        public double Longitude { get; private set; }
        
        public Collection<Line> Lines { get; set; }

        public Stop() { }
        
        public Stop(string name, double latitude, double longitude, Collection<Line> lines)
        {
            Name = name;
            Latitude = latitude;
            Longitude = longitude;
            Lines = lines;
        }
    }
}